vivarium/core/emitter.py

```diff
@@ -306,6 +307,7 @@ class DatabaseEmitter(Emitter):
     >>> # emitter = DatabaseEmitter(config)
     """
     default_host = 'localhost:27017'
+    client_dict: Dict[int, MongoClient] = {}
```

Comment:
Should this be passed in? How is the user expected to use this?

Reply:
The user is not expected to use this at all. Creating and managing `MongoClients` should occur seamlessly behind the scenes the same way we previously handled the singleton `MongoClient`. See [here](https://github.com/vivarium-collective/vivarium-core/blob/c5220e721fc96116dcd2fcb8bb94a86604f804b8/vivarium/core/emitter.py#L314).

Reply:
I’ll update the code to tie `client_dict` to the class instead of the instance.

Reply:
got it! looks good.